src/layouts/DefaultLayout.tsx

```diff
@@ -64,6 +64,17 @@ export const Layout = ({ children, ...props }: Props) => {
         },
       ],
     );
+
+    if (["TALLYING" || "RESULTS"].includes(roundState!)) {
```

Comment:
I think it could actually be useful to be able to creating the pool at all times during the round state.

Reply:
Will discuss and open up a new PR to handle this if folks agree